client/src/components/RelatedProducts/Outfits/CarouselOutfit.jsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable max-len */
+import { React, useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import { FiChevronLeft, FiChevronRight } from 'react-icons/fi';
+import { Individualcard } from '../Cards/Individualcard';
+import { useCurrentProductId } from '../../../contexts/ProductIDContext';
+import { useMeta } from '../../../contexts/ReviewMeta';
+import { useCurrentStyles } from '../../../contexts/StylesProvider';
+import useTracking from '@Contexts/ClickTracker';
+
+function CarouselAddToOutfit({ informationArray }) {
+  const { trackEvent } = useTracking({ widget: 'Add to Outfit'});
+  const [viewIndex, setViewIndex] = useState(0);
+  const { currentProductId } = useCurrentProductId();
+  const currentStyles = useCurrentStyles();
+  const starRating = useMeta();
+  const displayed = informationArray.slice(viewIndex, (viewIndex + 3));
+  const maxDisplayed = informationArray.length - 3;
+
+
+  const next = () => {
+    if (viewIndex === maxDisplayed) {
+      setViewIndex((viewIndex) => viewIndex - 1);
+    }
+      setViewIndex((viewIndex) => viewIndex + 1);
+  };
+
+  const prev = () => {
+    if (viewIndex === 0) {
+      setViewIndex(0);
+    } else {
+      setViewIndex((viewIndex) => viewIndex - 1);
+    }
+  };
+
+  const AddToStorage = (e, product) => {
```

Comment:
More naming convention stuff - I'd keep this in camel case `addToStorage` so you don't mix it up with a component by mistake